### PR TITLE
Backwards Compat on the exporter 

### PR DESCRIPTION
**Pull request by cklosowski**
#52 #44

### CODE REVIEW

File: includes/admin/list.php [hunk 1]
```diff
@@ -79,6 +80,37 @@ function edd_commissions_page() {
 				</form>
 
 			</div>
+			<?php endif; ?>
+
+			<?php if ( defined( 'EDD_VERSION' ) && version_compare( '2.4.2', EDD_VERSION, '>' ) ) : ?>
```
Comment (pippinsplugins) [inline includes/admin/list.php]: Why is this not an else statement?

File: includes/EDD_C_List_Table.php [hunk 2]
```diff
@@ -154,7 +154,7 @@ function get_paged() {
 	 * @return      mixed Int if user ID, string if email or login
 	 */
 	function get_filtered_user() {
-		$user_id = isset( $_GET['user'] ) ? sanitize_text_field( $_GET['user'] ) : 0;
+		$user_id = isset( $_GET['user'] ) && ! empty( $_GET['user'] ) ? sanitize_text_field( $_GET['user'] ) : 0;
```
Comment (pippinsplugins) [inline includes/EDD_C_List_Table.php]: This could be just `empty()`. Using `isset()` and `empty()` is redundant.

Comment (cklosowski) [inline includes/EDD_C_List_Table.php]: :+1:

File: includes/admin/list.php (hunk 1 above)
Comment (cklosowski) [inline includes/admin/list.php]: It was originally written with elements in between them, so I needed two different checks. Will update.

**Pull request merged**